setup-express.js

```diff
@@ -0,0 +1,119 @@
+const expressWinston = require('express-winston');
```

Comment:
Line space please

Reply:
lol..

Reply:
And also rename them to not include the setup prefix.